src/test/kotlin/uk/gov/justice/digital/hmpps/managerecallsapi/component/AssignRecallComponentTest.kt

```diff
@@ -52,7 +54,7 @@ class AssignRecallComponentTest : ComponentTestBase() {
     setupUserDetailsFor(assignee)
     setupUserDetailsFor(createdByUserId)
 
-    val recall = Recall(recallId, nomsNumber, createdByUserId, now, FirstName("Barrie"), null, LastName("Badger"))
+    val recall = Recall(recallId, nomsNumber, createdByUserId, now, FirstName("Barrie"), null, LastName("Badger"), CroNumber("ABC/1234A"), LocalDate.of(1999, 12, 1))
```

Comment:
I'm not a fan of re-using the same data, e.g. `CroNumber("ABC/1234A"), LocalDate.of(1999, 12, 1`) - nor of using it literally rather than via a reference or a `random` isation .... for so many tests but I guess it is the "Barrie Badger" way!